hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -676,6 +677,60 @@
 
 
     /**
+     * Type of Query Index result copying behavior.
+     *
+     * Defines the behavior for index copying on index read/write.
+     *
+     * Supported in BINARY, OBJECT and NATIVE in-memory-formats.
```

Comment:
Supported for BINARY and OBJECT in-memory formats. Only in Hazelcast 3.8.7, it is also supported for NATIVE in-memory format.